DTM Contextual help contents (#725)
SoftwareAG/c8y-docs: content/dtm/localization.md

weight: 40
layout: bundle
title: Localization
outputs:
- html
- json
helpcontent:
- label: localization
  title: Localization
  content: "

>   If you want to add translations for the custom properties or asset types created, or if you want to add custom translations for the existing static text in the DTM application, you can achieve it by using the Localization feature. 

Author: included the suggestion in latest commit

>   Once you create all custom properties and asset types, all those names are added as keys in the **Localization** page. You can further add the translations as required. 

Reviewer: ```suggestion
Once you created all custom properties and asset types, their names are added as keys in the **Localization** page. Add further translations as required. 
```

Author: included the suggestion in latest commit

>   To add a new translation in the DTM application, click **Add term** in the top menu bar. Add the text to be translated as key and add the translations as required. 

Reviewer: ```suggestion
  To add a new translation, navigate to the **Localization** menu in the navigator. On the **Localization** page, click **Add term** on the top right. In the resulting dialog box, add the name of the key, followed by the translation in the respective field.

Author: Because the user is already in the localization page while viewing this content, modified the suggestion as required.

>   *To use this feature, subscribe to Feature branding in the Enterprise tenant.*"

Reviewer: ```suggestion
Note that to use this feature, you must be subscribed to the feature Branding in the Enterprise tenant."
```

Author: Because the subscription is needed for Feature-branding application, modified the suggestion as required.
